tests/tests/worm-xattr-unit.sh

```diff
@@ -0,0 +1,78 @@
+t_require_commands touch rm setfattr
+
+touch "$T_D0/file-1"
+SECS=$(date '+%s')
+NSECS=$(date '+%N')
+DELAY=10
+EXP=$(( $SECS + $DELAY ))
+EXP_DELAY=$(( $DELAY + 2 ))
+
+echo "== test creation of worm xattr without .hide"
+setfattr -n scoutfs.worm.v1_expiration -v $EXP.$NSECS "$T_D0/file-1"
+
+echo "== test creation of worm xattr"
+setfattr -n scoutfs.hide.worm.v1_expiration -v $EXP.$NSECS "$T_D0/file-1"
+
+echo "== test value is set correctly"
+diff -u <(echo "$EXP.$NSECS") <(getfattr --absolute-names --only-values -n scoutfs.hide.worm.v1_expiration -m - "$T_D0/file-1" | grep $EXP.$NSECS)
```

Comment:
We don't want the grep here, we want the second "file" to be the raw getattr output so that we can see it in the diff output if it doesn't match what we set.